dts/arm/nxp/nxp_rt.dtsi

```diff
@@ -656,6 +656,7 @@
 			#size-cells = <0>;
 		};
 
+
```

Comment:
We can delete this extra line that was added

Reply:
ok